setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name="insightconnect_integrations_validators",
-    version="2.40.0",
+    version="2.41.0",
```

Comment:
```suggestion
    version="2.42.0",
```
Need to version bump after supported versions validator release.